packages/mirrorful/editor/src/components/Layout.tsx

```diff
@@ -0,0 +1,136 @@
+import { Box, Spinner, useDisclosure } from '@chakra-ui/react'
```

Comment:
Intentional that `Layout.tsx` is duplicated in `packages/mirrorful` and `apps/web`?